src/openforms/js/components/admin/form_design/variables/prefill/klantinteracties/KlantinteractiesFields.js

```diff
@@ -0,0 +1,50 @@
+import {useFormikContext} from 'formik';
+import {useContext, useEffect} from 'react';
+
+import Fieldset from 'components/admin/forms/Fieldset';
+import ValidationErrorsProvider, {
+  ValidationErrorContext,
+} from 'components/admin/forms/ValidationErrors';
+import ErrorMessage from 'components/errors/ErrorMessage';
+
+import EmailField from './EmailField';
+import PhoneNumberField from './PhoneNumberField';
+
+const KlantinteractiesFields = () => {
+  const {values, setFieldValue} = useFormikContext();
+  const errors = Object.fromEntries(useContext(ValidationErrorContext));
+  const optionsErrors = Object.entries(errors.options ?? {}).map(([key, errs]) => [
+    `options.${key}`,
+    errs,
+  ]);
+  const nonFieldErrors = optionsErrors.filter(([key]) => key === 'options.nonFieldErrors');
+
+  const defaults = {
+    email: false,
+    phoneNumber: false,
+  };
```

Comment:
can be defined outside of the component implementation, because it's invariant

Reply:
defined defaults outside of the component